### ISSUE TITLE
Support for browsers other than Google Chrome

### ISSUE THREAD

**Issue by RdkLwg** [Cognifide/aet#513]
Prepare a PoC and a manual how to make use of browsers different than Google Chrome.

**Comment by tMaxx** [comment]
See #471.

In there, the argumentation is that AETs are not a tool for cross-browser testing. Is this still up-to-date?

**Comment by RdkLwg** [comment]
Yeah, the idea is that AET is not a cross-browser testing tool nor the tool to compare a site among different browsers, however one may need to track changes in a browser different than Chrome (e.g. MS Edge or Mozilla Firefox because it is the main browser used in the project or a given browser may be prone to regression changes than others).

So we need 
1. support for:
* Mozilla Firefox
* MS Edge
2. and a feature allowing the user to choose a browser when running a test (e.g. with additional flag/paramater).
